### PR TITLE
EN-76: Prefill new contract/assignment with active information

### PR DESCRIPTION
**Pull request by FacuOrellana**
# Description :pencil:

When a new contract/employee is created from employees profile, the latest active information is prefill in the form.

## Link to ticket :link:

https://entropyteam.atlassian.net/browse/EN-76

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? :microscope:

Manual testing
New contract prefill with information from active contract
image

New contract prefill from employee without active contract
image

New assignment prefill with information from latest employee assignment
image

New assignment from employee without assignment
image

### CODE REVIEW

**Review by agustinb4 — CHANGES_REQUESTED** [review]

File: src/employeeProfiles.js [hunk 1]
```diff
@@ -33,6 +34,27 @@ const DisplayRecordCurrentId = () => {
...
+  return data !== undefined ? data.at(0) : null;
+};
+
+const GetLatestAssignment = () => {
+  const employeeId = useGetRecordId();
+  const { data } = useGetManyReference("assignments", {
+    target: "employeeId",
+    id: employeeId,
+  });
+  return data !== undefined ? data.find((a) => a.endDate === null) : null;
```
[the remark this answers is not in the document]
Comment (FacuOrellana) [reply src/employeeProfiles.js:55]: in that case, witch assignment we should consider as lastest?

[the remark this answers is not in the document]
Comment (agustinb4) [reply src/employeeProfiles.js:55]: I guess we could filter for the assignment with the last start dale. 
Also we have to contemplate the case when we have 2 active assignment. so it should work
what do you think?

[the remark this answers is not in the document]
Comment (FacuOrellana) [reply src/employeeProfiles.js:55]: it should work, we have to keep the same rule to display the current assignment in employee card. Until we add active field in assignment we should use the same criteria in both places.

[the remark this answers is not in the document]
Comment (agustinb4) [reply src/employeeProfiles.js:55]: you're right. for that case i used the last end date when all the assignments has end date. 
what's about add the last assignment to the employee dto, then use this data to populate de assignments fields?

[the remark this answers is not in the document]
Comment (FacuOrellana) [reply src/employeeProfiles.js:55]: sounds like a great solution to me

**Review by baremarco — APPROVED** [review]

**Review by agustinb4 — APPROVED** [review]

**Pull request merged**